packages/react-components/src/components/graph/graph.tsx

```diff
@@ -1,40 +1,78 @@
-import React, { FC, HTMLAttributes, useCallback, useEffect, useRef } from 'react';
-import type { Core, ElementDefinition, LayoutOptions } from 'cytoscape';
-import { FitIcon, MinusIcon, PlusIcon, TargetIcon } from './lib/svgs/icons';
-import Container from '@awsui/components-react/container';
+import React, { HTMLAttributes, useCallback, useEffect, useRef } from 'react';
+import type { Core, ElementDefinition, EventObject } from 'cytoscape';
+import { response3 as response } from './lib/response';
 
-import CytoscapeComponent from 'react-cytoscapejs';
+import useCyEvent from './hooks/useCyEvent';
+import GraphView from './graph-view';
+import Toolbar from './graph-toolbar';
 
 import './styles.scss';
+import { Button } from '@cloudscape-design/components';
+import { STYLE_PREFIX } from './constants';
+import useStylesheet from './cytoscape-cloudscape-theme';
 
-export interface CytoGraphProps extends HTMLAttributes<HTMLDivElement> {
-  graphLayout?: LayoutOptions;
+export interface GraphProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
In my next PR I will add more to the props like nodes and edges and all props we might need to expose. I want to move the logic of parsing outside the graph components.